src/symposium-app/symposium-app.html

```diff
@@ -110,11 +117,11 @@
         </app-toolbar>
 
         <!-- Nav on mobile: side nav menu -->
-        <paper-menu selected="{{selected}}" on-tap="scrollTop">
-            <paper-item>Home </paper-item>
-            <paper-item>Program</paper-item>
-            <paper-item>Location</paper-item>
-            <paper-item>Committee</paper-item>
+        <paper-menu selected="{{data.page}}" attr-for-selected="key" on-tap="scrollTop">
+            <paper-item key="">Home</paper-item>
```

Comment:
Yes. See https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes